cmd/git-sync/main.go

```diff
@@ -2111,6 +2150,11 @@ OPTIONS
             The git submodule behavior: one of "recursive", "shallow", or
             "off".  If not specified, this defaults to "recursive".
 
+    --sync-on-signal <string>, $GIT_SYNC_SYNC_ON_SIGNAL
+            Indicates that an immediate sync should occur upon receipt of the 
+            specified signal name (e.g. SIGHUP) or number (e.g. 1). If not 
```

Comment:
for clarity, something like:

"""
Indicates that a sync attempt should occur upon receipt of the specified signal name (e.g. SIGHUP) or number (e.g. 1).  If a sync is already in progress, another sync will be triggered as soon as the current one completes.  If not specified...
"""

The key delta being "immediate" semantics.